docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -127,21 +129,30 @@ Using GPUs with PyTorch
 You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery.
 
 The following is an example of using a conda virtual environment with PyTorch for CUDA version 11.1. Make sure that you are on a GPU node before loading the environment::
-
+  
+  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
   module load cuda/11.1
   module load anaconda3/2021.11
   conda create --name pytorch_env python=3.7 -y
   source activate pytorch_env
   conda install pytorch==1.8.0 torchvision==0.9.0 torchaudio==0.8.0 cudatoolkit=11.1 -c pytorch -c conda-forge -y
```

Comment:
I was able to get this working in a slightly different manner and without having to provide individual versions.

Reply:
@manasvita added lightweight and heavyweight Pytorch installation versions.